app/src/main/java/org/dhis2/utils/analytics/AnalyticsInterceptor.kt

```diff
@@ -40,15 +40,15 @@ class AnalyticsInterceptor(private val analyticHelper: AnalyticsHelper) : Interc
         val request = chain.request()
         val response = chain.proceed(request)
 
-        if (response.code() >= 400 && !isLogged) {
+        if (response.code >= 400 && !isLogged) {
             isLogged = D2Manager.getD2().userModule().blockingIsLogged()
         }
```

Comment:
I think this was changed in another PR, could cause conflicts

Reply:
it is the same... it's just code is a variable not a method